app/bundles/ReportBundle/Assets/js/report.js

```diff
@@ -216,18 +216,26 @@ Mautic.updateReportFilterValueInput = function (filterColumn, setup) {
     Mautic.destroyChosen(mQuery('#' + valueId));
 
     if (filterType == 'bool' || filterType == 'boolean') {
-        if (mQuery(valueEl).attr('type') != 'radio') {
-            var template = mQuery('#filterValueYesNoTemplate .btn-group').clone(true);
-            mQuery(template).find('input[type="radio"]').each(function () {
-                mQuery(this).attr('name', valueName);
-                var radioVal = mQuery(this).val();
-                mQuery(this).attr('id', valueId + '_' + radioVal);
-            });
-            mQuery(valueEl).replaceWith(template);
-        }
-
-        if (setup) {
-            mQuery('#' + valueId + '_' + valueVal).click();
+        const yesId = valueId + '_1';
+        const noId = valueId + '_0';
+        const isYes = valueVal == '1';
+        const $template = mQuery('#filterValueYesNoTemplate');
```

Comment:
```suggestion
        const $template = mQuery('#filterValueYesNoTemplate').clone();
        $template.attr('id', valueId + '_bool-container');
```
This is causing problem with creating the new filter element.